d/dotnet7/prereq_install

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+apt install -y dpkg-dev
+
+dpkg --add-architecture ppc64el
+sudo apt-get update
+
+apt install -y crossbuild-essential-ppc64el
```

Comment:
this can be a single apt install command with full dependency list, that'll make the script cleaner?

Reply:
yes changed it to single apt install